.travis.yml

```diff
@@ -21,33 +21,49 @@ services:
   - docker
 
 before_install:
-  # Use OS package to update JDK
-  - OPENJDK_VERSION=11
-  - sudo apt-get -y install openjdk-${OPENJDK_VERSION}-jdk
-  - export JAVA_HOME=/usr/lib/jvm/java-${OPENJDK_VERSION}-openjdk-amd64/
-  - ./mvnw -version
+  - |
+    # Use JDK 11, so we can release Java 6 bytecode
+    OPENJDK_VERSION=11
+    sudo apt-get -y install openjdk-${OPENJDK_VERSION}-jdk
+    export JAVA_HOME=/usr/lib/jvm/java-${OPENJDK_VERSION}-openjdk-amd64/
+    ./mvnw -version
 
-  # Required for Elasticsearch 5 (See https://github.com/docker-library/docs/tree/master/elasticsearch#host-setup)
-  - sudo sysctl -w vm.max_map_count=262144
+    # Required for Elasticsearch 5 (See https://github.com/docker-library/docs/tree/master/elasticsearch#host-setup)
+    sudo sysctl -w vm.max_map_count=262144
 
-  # Quiet Maven invoker logs (Downloading... when running zipkin-server/src/it)
-  - echo "MAVEN_OPTS='-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'" > ~/.mavenrc
+    # Quiet Maven invoker logs (Downloading... when running zipkin-server/src/it)
+    echo "MAVEN_OPTS='-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'" > ~/.mavenrc
 
-  # Disable testcontainers checks
-  - echo checks.disable=true > ~/.testcontainers.properties
-  # Log in to Docker Hub for releasing the image
-  - echo "$GH_TOKEN"| docker login ghcr.io -u "$GH_USER" --password-stdin
-  # allocate commits to CI, not the owner of the deploy key
-  - git config user.name "zipkinci"
-  - git config user.email "zipkinci+zipkin-dev@googlegroups.com"
-  # setup https authentication credentials, used by ./mvnw release:prepare
-  - git config credential.helper "store --file=.git/credentials"
-  - echo "https://$GH_TOKEN:@github.com" > .git/credentials
+    # Disable testcontainers checks
+    echo checks.disable=true > ~/.testcontainers.properties
 
-# Override default travis to use the maven wrapper
-# * Use the "release" profile as that ensures the core jar ends up JRE 1.6 compatible
-install: ./mvnw install -DskipTests=true -Dlicense.skip=true -Dmaven.javadoc.skip=true -B -V -Prelease
-script: ./travis/publish.sh
+    # Credentials entered into https://travis-ci.org/github/openzipkin/${REPO}/settings are access
```

Comment:
note: this is currently lying as I've not removed them from the bottom of the yaml, yet. Just yeah..